src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/FallThroughCheck.java

```diff
@@ -102,17 +64,118 @@
  * To configure the check:
  * </p>
  * <pre>
- * &lt;module name=&quot;FallThrough&quot;/&gt;
+ * &lt;module name="FallThrough"/&gt;
+ * </pre>
+ * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * switch (i) {
+ *   case 1:
+ *     i++;
+ *   case 2: // violation, case 1 contains code but lacks break, return, throw or continue statement
+ *     i++;
+ *     break;
+ *   case 3: // OK
+ *     i++;
+ *     return;
+ *   case 4: // OK
+ *     i++;
+ *     throw new Exception();
+ *   case 5: // OK
+ *     i++;
+ *     continue;
+ *   case 6: // OK
+ *   case 7: // OK, case 6 does not contain code
+ *     i++;
+ *   // OK, by default the last case is not checked
+ * }
+ * </pre>
+ * <p>
+ * Unreachable code still causes violation:
+ * </p>
+ * <pre>
+ * switch (i) {
+ *   default:
+ *     return;
+ *   case 1:
+ *     i++; // unreachable
+ *   case 2: // violation
+ *     return;
+ * }
  * </pre>
  * <p>
- * or
+ * Add special comments to suppress the warning:
+ * </p>
+ * <pre>
+ * switch (i) {
+ *   case 1:
+ *     i++; // fall through
+ *
+ *   case 2: // OK
+ *     i++;
+ *     // fallthru
+ *   case 3: { // OK
+ *     i++;
+ *   }
+ *   &#47;* fall-thru *&#47;
+ *   case 4: // OK
+ *   i++;
+ *     // Fallthru
+ *   case 5: // violation, "Fallthru" in case 4 should be "fallthru"
+ *     i++;
+ *     // fall through
+ *     i++;
+ *   case 6: // violation, the comment must be on the last non-empty line before 'case'
+ *     i++;
+ *   &#47;* fall through *&#47;case 7: // OK, comment can appear on the same line but before 'case'
+ *     i++;
+ * }
+ * </pre>
+ * <p>
+ * To configure the check so that it checks last case group
```

Comment:
1. `so that it checks last case group` - > to enable check for last case group
2. Add colon at the end

Reply:
I'll fix that

Reply:
fixed